src/test/conversion/options/Option.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { JavaToTypescriptConverter } from "../../../conversion/JavaToTypeScript";
+import fs from 'fs/promises';
+
+export {}
```

Comment:
That `export {}` can be removed. I added that only to spec files which have no tests yet and are not considered being modules without an import or export.